includes/deprecated/deprecated-functions.php

```diff
@@ -641,4 +641,39 @@ function give_build_paypal_item_title( $payment_data ) {
 
 	return give_payment_gateway_item_title( $payment_data );
 
+}
+
+
+/**
+ * Set the number of decimal places per currency
+ *
+ * @since 1.0
+ * @since 1.6 $decimals parameter removed from function params
+ * @deprecated 1.8.15
+ * *
+ * @return int $decimals
+ */
+function give_currency_decimal_filter() {
```

Comment:
@DevinWalker This function is deprecated. I also removed filter because `give_get_price_decimals` will handle this internally.